tests/system/model_monitoring/test_app.py

```diff
@@ -178,6 +187,7 @@ def _test_v3io_records(
         cls._test_apps_parquet(ep_id, inputs, outputs)
         cls._test_kv_record(ep_id)
         cls._test_tsdb_record(ep_id)
+        cls._test_predictions_table(ep_id)
 
     @classmethod
     def _test_api_get_results(
```

Comment:
I would also like to see tests for this metric in the `_test_api` method.
It's not covered despite the central role it has

Reply:
Still working on it, due to the long testing cycle.